test_ffi/tests/bench.js

```diff
@@ -0,0 +1,504 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+// deno-lint-ignore-file
+
+//const targetDir = Deno.execPath().replace(/[^\/\\]+$/, "");
+const targetDir = "/home/tiffany/dev/rust/deno/target/debug";
```

Comment:
should be pointing to a release build instead?

Reply:
This... shouldn't be here. The above `execPath()` is the proper way to get it, I think.

Reply:
Removed the explicit path.